src/views/user/Login.vue

```diff
@@ -4,13 +4,15 @@
       <b-row align-h="center" class="mt-5">
         <b-col cols="12" md="6">
           <b-card class="p-3">
-            <h3 class="mb-4">Log In</h3>
+            <h2 class="mb-4 text-center">Log In</h2>
+            <div style="text-align:center">
+              <a href="/register">or create an account</a>
+            </div>
             <b-form @submit.prevent="login" v-if="show" :novalidate="true">
```

Comment:
here `v-if="show"` is not needed, you can just remove it and also remove property `show: true` from data.